md/release-notes.md

```diff
@@ -3,100 +3,114 @@
 Bonita BPM 7.4 Beta includes a set of user-based product enhancements, bug fixes related to Bonita BPM Engine, 
 a new REST connector and important technology updates.
 
-## user-based product enhancements
+## User-based product enhancements
+
+<a id="bpm-studio"/>
+
+### Bonita BPM Studio
+* When importing a .bos file, 3 options are now available:
+    * import it in the current repository
+    * import it in another existing repository
+    * create a new repository and import it there
+  This may prevent from overriding a BDM file or other types of development artifacts.
+* Contract inputs can be defined with the type LONG, at pool level only. In called processes, this is helpful to receive Ids from call activities
+* The list of operations in a task and in the outputs of a connector configuration can be reordered
+* Process simulation capability has been removed from the product `(removed)`
 
 <a id="bdm-usability-improvements"/>
 
-### BDM
-* .xml is now the storage format in the Studio repository making it possible using any VCS to handle a diff of 2 different revisions of the same BDM
-* in the Studio, a new menu gives access to the H2 console to visualize and browse business data
-* when importing a .bos file in the Studio it is possible to select another repository in order to manage resources from multiple projects `(SP)`
+### Business Data Model (BDM)
+* In the Studio, a new menu item Development > Business Data Model > Browse data (h2 console) opens the H2 console to visualize and browse business data
```

Comment:
I would add: This is useful to see the instances of you Business Objects that while testing your process from the Studio environment